plugins/pulp_rpm/plugins/migrations/0020_nested_drpm_directory.py

```diff
@@ -0,0 +1,119 @@
+import glob
+import logging
+import os
+import shutil
+import tempfile
+
+import lxml.etree as le
+
+from pulp.server.config import config
+from pulp.server.db.connection import get_collection
+
+
+_logger = logging.getLogger('pulp_rpm.plugins.migrations.0020')
```

Comment:
You can use ```__file__``` instead of pulp_rpm.plugins.migrations.0020, and it will automatically reference the correct module, even if this eventually gets renamed.